Make install failing with Serde trait bound errors

`make install` fails with the following:

![image](https://user-images.githubusercontent.com/1276854/70369339-bf5e4500-18f2-11ea-8914-4117abce55f4.png)


```
rustup run --install 1.39.0 cargo --color always install --force --path cnd
  Installing cnd v0.5.0 (/home/farlion/code/tenx/comit-rs/cnd)
    Updating crates.io index
   Compiling cnd v0.5.0 (/home/farlion/code/tenx/comit-rs/cnd)
error[E0277]: the trait bound `bitcoin::blockdata::transaction::Transaction: config::_IMPL_SERIALIZE_FOR_Data::_serde::Serialize` is not satisfied
  --> cnd/src/swap_protocols/ledger/bitcoin.rs:23:6
   |
23 | impl Ledger for Bitcoin {
   |      ^^^^^^ the trait `config::_IMPL_SERIALIZE_FOR_Data::_serde::Serialize` is not implemented for `bitcoin::blockdata::transaction::Transaction`

error[E0277]: the trait bound `for<'de> bitcoin::blockdata::transaction::Transaction: config::_IMPL_SERIALIZE_FOR_Data::_serde::Deserialize<'de>` is not satisfied
  --> cnd/src/swap_protocols/ledger/bitcoin.rs:23:6
   |
23 | impl Ledger for Bitcoin {
   |      ^^^^^^ the trait `for<'de> config::_IMPL_SERIALIZE_FOR_Data::_serde::Deserialize<'de>` is not implemented for `bitcoin::blockdata::transaction::Transaction`

error[E0277]: the trait bound `bitcoin::blockdata::transaction::OutPoint: config::_IMPL_SERIALIZE_FOR_Data::_serde::Serialize` is not satisfied
  --> cnd/src/swap_protocols/rfc003/bitcoin/mod.rs:16:6
   |
16 | impl Ledger for Bitcoin {
   |      ^^^^^^ the trait `config::_IMPL_SERIALIZE_FOR_Data::_serde::Serialize` is not implemented for `bitcoin::blockdata::transaction::OutPoint`

error[E0277]: the trait bound `for<'de> bitcoin::blockdata::transaction::OutPoint: config::_IMPL_SERIALIZE_FOR_Data::_serde::Deserialize<'de>` is not satisfied
  --> cnd/src/swap_protocols/rfc003/bitcoin/mod.rs:16:6
   |
16 | impl Ledger for Bitcoin {
   |      ^^^^^^ the trait `for<'de> config::_IMPL_SERIALIZE_FOR_Data::_serde::Deserialize<'de>` is not implemented for `bitcoin::blockdata::transaction::OutPoint`

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0277`.
error: failed to compile `cnd v0.5.0 (/home/farlion/code/tenx/comit-rs/cnd)`, intermediate artifacts can be found at `/home/farlion/code/tenx/comit-rs/target`

Caused by:
  could not compile `cnd`.

To learn more, run the command again with --verbose.
Makefile:40: recipe for target 'install' failed
make: *** [install] Error 101
```

--- a/cnd/Cargo.toml
+++ b/cnd/Cargo.toml
@@ -11,7 +11,7 @@ async-std = { version = "1", features = ["unstable"] }
 async-trait = "0.1"
 bigdecimal = "0.1.0"
 binary_macros = "0.6"
-bitcoin = "0.19.1"
+bitcoin = { version = "0.19.1", features = ["use-serde"] }
 blockchain_contracts = "0.1"
 byteorder = "1.3"
 chrono = { version = "0.4", features = ["serde"] }

